schema.yml

```diff
@@ -80,9 +80,7 @@ mapping:
 
           Environment variables with only a key are resolved to their values on the machine Compose is running on, which
```

Comment:
good catch, missed that. I will update it to say "You can use an array or a hash."